app/models/labware_creators/common_file_handling/csv_file/row_base.rb

```diff
@@ -86,9 +86,7 @@ def check_for_invalid_characters
       private
 
       def cell_is_blank?(cell)
-        # NB. cannot use cell.blank? as triggers an exception if cell contains the unusual characters
-        # are trying to spot
-        cell.nil? || cell.strip.empty?
+        cell.nil?
```

Comment:
The cell.strip.empty? was to check for cells with just spaces in them (and possibly invisible characters). This change lets them through I think. 
There were RTs where they'd copy pasted invalid for UTF8 characters into cells. blank? was blowing up for the weird characters, but we need the equivalent check.
empty? is not the same as nil?
Or you think that case is covered?

Reply:
sorry, strip.empty? is not the same as nil? is what I meant
cells with only spaces in them need to be treated as if empty/nil

Reply:
Given that you seem to have proven that the behavior is correct (or have I got the wrong end of the stick?), it seems now that youhave effectively created an alias method for `cell.nil?` so would it be worth inlining this method where it's used?  i.e. where calling `cell_is_blank?(value)` instead call `value.nil?`.

Reply:
Yes looks good thanks